modules/virt-enabling-load-balancer-service-web.adoc

```diff
@@ -18,4 +18,4 @@ You can enable the creation of load balancer services for a virtual machine (VM)
 
 . Navigate to *Virtualization* -> *Overview*.
 . On the *Settings* tab, click *Cluster*.
-. Expand *LoadBalancer service* and select *Enable the creation of LoadBalancer services for SSH connections to VirtualMachines*.
\ No newline at end of file
+. Expand *General settings* and *SSH configuration*, and enable *SSH over LoadBalancer service*.
```

Comment:
OK, now I see that you are documenting the 4.15 UI. I would break this into 2 sentences because you have 3 steps here.

Suggestion:

. Expand *General settings* and *SSH configuration*.
. Set *SSH over LoadBalancer service* to on. 
                                                                          ^^^ No markup
                                                                          
I know it looks weird to see "on" without bold but that's what ISG says. See the [UI guidelines](https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=CCSENABLE&title=UI+guidelines%3A+Quick+reference) or ISG. 

We don't use the word "enable" for the UI because it is ambiguous. A switch is either on or off. If the switch does not have a label that says "On", then the state ("on") is lower-case and not bolded. Hope that makes sense!

I noticed that 4.15 also has a NodePort service switch. I need to check for a task for that.